ui/webui/src/components/time/DateAndTime.jsx

```diff
@@ -0,0 +1,505 @@
+                                      fieldId="time-date-edit-date"
+                                    >
+                                        <DatePicker
+                                          isDisabled={NTPSyncEnabled}
+                                          dateFormat={timeformat.date}
+                                          dateParse={timeformat.parseDate}
+                                          value={timeformat.date(systemDate.valueOf())}
+                                          onChange={handleDateChange}
+                                        />
+                                    </FormGroup>
+                                    <FormGroup
+                                      label={_("Time")}
+                                      isRequired
+                                      fieldId="time-date-edit-time"
+                                    >
+                                        <TimePicker
+                                          isDisabled={NTPSyncEnabled}
+                                          time={systemDate}
+                                          onChange={handleTimeChange}
+                                        />
+                                    </FormGroup>
+                                </Flex>
+                            </GridItem>
+                        </Grid>
+                    </FormSection>
+                    <Divider />
+                    <Text className="time-edit-automatic-hint">
+                        {_("To edit date and time, turn off automatic time")}
+                    </Text>
+                    <ActionGroup>
+                        <Button
+                          id={idPrefix + "-btn-save"}
+                          variant="primary"
+                          onClick={handleSave}
+                        >
+                            {_("Save")}
+                        </Button>
+                        <Button
+                          id={idPrefix + "-btn-cancel"}
+                          variant="link"
+                          onClick={handleCancel}
+                        >
+                            Cancel
+                        </Button>
+                    </ActionGroup>
+                </Form>
+            </CardBody>
+        </Card>
+    );
+};
+
+const AutomaticTimeNotification = ({ automaticTimeEnabled }) => {
+    return (
+        <Flex direction={{ default: "row" }} spaceItems={{ default: "spaceItemsSm" }} alignSelf={{ default: "alignItemsCenter" }}>
+            <FlexItem>
+                <Icon status="info">
+                    <InfoCircleIcon />
+                </Icon>
+            </FlexItem>
+            <FlexItem>
+                <Text className="automatic-time-label">
+                    {automaticTimeEnabled ? _("Automatic time is on.") : _("Automatic time is off.")}
+                </Text>
+            </FlexItem>
+            <FlexItem>
+                <Popover
```

Comment:
The helper icon has somehow different color in the mockup. Looks like more like the variant from https://www.patternfly.org/v4/components/form#basic

Try copying that one.